### PR TITLE
block hawkes_cumulant test until tf v2 is supported

### PR DESCRIPTION
**Pull request by Dekken**

### CODE REVIEW

File: tick/hawkes/inference/hawkes_cumulant_matching.py [hunk 1]
```diff
@@ -279,6 +279,7 @@ def _tf_placeholders(self):
         d = self.n_nodes
         if self._tf_feed_dict is None:
             with self._tf_graph.as_default():
+                tf.compat.v1.disable_eager_execution()
```
Comment (sara-02) [inline tick/hawkes/inference/hawkes_cumulant_matching.py]: Along with this the `tf.graph` initialization might also require a change:
https://github.com/X-DataInitiative/tick/blob/master/tick/hawkes/inference/hawkes_cumulant_matching.py#L166 
As listed there: https://www.tensorflow.org/guide/migrate#top-level_behavioral_changes

Comment (Dekken) [reply tick/hawkes/inference/hawkes_cumulant_matching.py]: that line that wasn't meant to be commit and has been removed

Comment (Dekken) [reply tick/hawkes/inference/hawkes_cumulant_matching.py]: that line wasn't meant to be commit and has been removed

File: tools/appveyor/run.sh [hunk 2]
```diff
@@ -6,20 +6,15 @@ CWD="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
-/c/Python37-x64/Scripts/pip install appveyor/pip/scipy-1.3.0-cp37-cp37m-win_amd64.whl
-/c/Python37-x64/Scripts/pip install -r requirements.txt
-/c/Python37-x64/Scripts/pip install tensorflow==1.14.0 psutil
+/c/Python36-x64/python -m pip install --upgrade pip
+/c/Python36-x64/Scripts/pip install -r requirements.txt
+/c/Python36-x64/Scripts/pip install psutil wheel
 
 set +e
-/c/Python37-x64/python setup.py build_ext -j 2 --inplace
+/c/Python36-x64/python setup.py build_ext -j 2 --inplace
```
Comment (sara-02) [inline tools/appveyor/run.sh]: @Dekken just curious why is the change from Python 3.7 to 3.6 required?

Comment (Dekken) [reply tools/appveyor/run.sh]: I'm testing appveyor to see what is causing the segfault

Comment (Dekken) [reply tools/appveyor/run.sh]: so far it seems python3.6 is ok but python3.7 is not

**Comment by Dekken** [comment]
fixes https://github.com/X-DataInitiative/tick/issues/411
       https://github.com/X-DataInitiative/tick/issues/408

**Pull request merged**